modules/javafx.controls/src/main/java/javafx/scene/control/skin/NestedTableColumnHeader.java

```diff
@@ -699,4 +715,86 @@ private void columnResizingComplete(TableColumnBase col, MouseEvent me) {
         columnReorderLine.setLayoutX(0.0F);
         lastX = 0.0F;
     }
+
+    /**
+     * Calculates and sets what width of the resize rectangle (rectangle between column
+     * headers) should size should be.
+     * @return
+     */
+    private void setResizeGrabRectangleWidth(Rectangle aRectangle, TableColumnHeader aTableColumnHeader) {
+
+        Insets headerPadding = aTableColumnHeader.getPadding();
+        double headerWidthWithoutPadding = aTableColumnHeader.getWidth() - headerPadding.getLeft() - headerPadding.getRight();
+        double headerHeight = aTableColumnHeader.getHeight();
+
+        Double childrenSumPrefWidth = aTableColumnHeader
```

Comment:
Have you tried to use `aTableColumnHeader.prefWidth(headerHeight)` to compute current pref height instead of sum children prefWidths ?

Reply:
yes, it's equals to the actual width which I'm not interested in

Reply:
have you tested this code with nested columns?